corehq/apps/builds/management/commands/commcare_build_importer.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import requests
+
+from cStringIO import StringIO
+from jenkinsapi.jenkins import Jenkins
+
+from corehq.apps.builds.models import CommCareBuild
+from django.core.management.base import BaseCommand, CommandError
+from dimagi.utils.decorators.memoized import memoized
+
+
+class Command(BaseCommand):
+    """
+    This is an automated version of -
+    https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/builds#adding-commcare-j2me-builds-to-commcare-hq
+    """
+    help = "Interactively import a build from Jenkins build server."
+
+    def handle(self, *args, **options):
+        if not args:
+            self.interactive_import()
+        else:
+            raise CommandError(self.help)
+
+    def interactive_import(self):
+
+        # supress requests library INFO logs
+        logging.getLogger("requests").setLevel(logging.WARNING)
+
+        # let user chose a project from the list ['commcare-mobile-2.21', ...]
+        selected_project_key = None
+        print "Jenkins has following projects. Please choose one (to end enter END)"
+        print [p for p in self.jenkin_projects if 'commcare-mobile' in p]
+        while selected_project_key not in self.jenkin_projects:
+            selected_project_key = raw_input("")
+            if selected_project_key.lower() == 'end':
+                return
```

Comment:
It'd be nice to give some feedback if the `selected_project_key` is not found - I typed `"commcare-mobile"` (with the quotes) and it just hung there, which was confusing